wiki/Tournaments/OIT/2023/en.md

```diff
@@ -136,6 +167,35 @@ Listed below are the players who managed to qualify into the Round of 16 (along
   2. [Tsukino - Dohna Dohna no Uta (gazimal) \[Koori's Insane\]](https://osu.ppy.sh/beatmapsets/1459224#osu/3082198)
   3. [Maon Kurosaki - Magic World (Dada) \[Continuum\]](https://osu.ppy.sh/beatmapsets/1833693#osu/3764472)
 
+## Match results
+
+| Lower | [Rexeez](https://osu.ppy.sh/users/1987591) ::{ flag=ID }:: | 3 | **6** | ::{ flag=ID }:: **[Vinno](https://osu.ppy.sh/users/10717635)** | [#1](https://osu.ppy.sh/community/matches/98853253) |
```

Comment:
something broke here

Reply:
oh yeah i had this as a reference but i forgot to delete this afterwards lol

should be fixed